docs/general/events.md

```diff
@@ -32,17 +32,73 @@ field semantics, and stability and requirement levels. Other events may be user-
+<!-- semconv event -->
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| `event.name` | string | Identifies the class / type of event. [1] | `browser.mouse.click`; `device.app.lifecycle` | `Required` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+**[1]:** Event names are subject to the same rules as [attribute names](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.31.0/specification/common/attribute-naming.md). Notably, event names are namespaced to avoid collisions and provide a clean separation of semantics for events in separate domains like browser, mobile, and kubernetes.
+<!-- endsemconv -->
+
+### General event semantics
+
+* An event MUST have an `event.name` attribute that uniquely identifies the event.
+* It MAY have other [standard](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/common#attribute)
+  attributes that provide additional context about the event.
+* It MAY contain a _payload_ (data) that describes the specific details of the
+  named event.
+* The structure of the named event (`event.name`) is unique to the event, this
+  identifies
+  * Whether the event contains a _payload_ (data), and the structure / type of
+    the _payload_ (data).
+  * Whether there are any expected [standard](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/common#attribute)
+    attributes that are used to describe the event.
+* All OpenTelemetry defined events SHOULD be defined in the semantic conventions
+  to highlight the expected structure of the event.
+* The _payload_ (data) MAY contain any type supported by the OpenTelemetry data
+  model for the log [body](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-body)
+  and the semantic conventions will define the expected structure of the _payload_
+  (data) for the event.
+* The _payload_ (data) SHOULD be used to represent the details.
+* Events with the same `event.name` are structurally similar to one another and
+  MAY (depending on the event) be comparable.
+
+Recommendations for defining events:
+
+* Use the _payload_ (data) to represent the details of the event instead of a
+  collection of  over [standard](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/common#attribute)
+  attributes.
+* Generators of events SHOULD use the [Event API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/event-api.md)
+  to create and emit events.
+  * While the data model is based on `LogRecord`, it should NOT be assumed that
+    ALL consumers will process events as `LogRecords`, some may choose to provide
+    their own Event API implementation.
```

Comment:
I don't understand what this sentence means in scope of semantic conventions, would it be possible to remove or clarity it?